### ISSUE TITLE
Ability to allow multiple destination

### ISSUE THREAD

**Issue by naisanza** [RequestPolicy/requestpolicy#411]
Similar to how NoScript works, where you can select several scripts to allow/block, and not until you move your cursor off the drop down menu, or click anywhere but the drop down, will it make changes and refresh the page. This is useful particularly for sites that have multiple various sources, and having the page refresh after setting each one is really slow and bothersome. 

Thanks for all the great work.

**Comment by naisanza** [comment]
Forgot to add labels. This files under Feature-request

**Comment by ghost** [comment]
This feature is already present in version 1.0.0b3.

**Comment by naisanza** [comment]
I'll have to check out that version!

**Comment by dfc** [comment]
I am not running the beta and was always annoyed by the click allow, reload, click allow, reload, click allow, reload dance. I just ended up disabling auto refresh and now I can click all the allows I want and then manually refresh. It is not a perfect solution but it works.

**Comment by naisanza** [comment]
I can verify that this works well in 1.0.0.b3, however, there's a good enhancement that I would like to add to the behavior of clicking off the app to remove focus back to the window (exactly like how NoScript handles this behavior, where you can click anything and it won't affect the window. e.g. links, buttons, etc). I'll add this to a new ticket requesting for enhancement.
